packages/server/test/unit/util/args_spec.js

```diff
@@ -229,6 +229,36 @@ describe('lib/util/args', () => {
     })
   })
 
+  context('--auto-cancel-after-failures', () => {
```

Comment:
Suggest adding a test for a negative number, a decimal number, and zero.

Reply:
the tests were added and the `0` value pass